index.js

```diff
@@ -295,13 +295,17 @@ export default class Measure extends Component {
    * @private
    */
   payload() {
-    const rendered = this.get('domContentLoaded');
-    const start = this.get('navigationStart');
+    const routeChangeToRenderMetrics = Measure.webVitals;
+    const rendered = this.get('domContentLoaded') && this.get('domContentLoaded').now ?
```

Comment:
I would highly suggest just caching the result of `this.get('domContentLoaded')` and all the `this.get` calls you're doing. You're doing 3 function calls when it should really just be 1. In addition to that, it will increase the readability of this.